packages/terra-horizontal-divider/src/HorizontalDivider.scss

```diff
@@ -0,0 +1,12 @@
+@import './variables';
+@import './mixins';
```

Comment:
Were the files these imports reference deleted?

Reply:
Yes the are deleted. https://github.com/cerner/terra-core/pull/962/commits/b6a950329f831a533a9f38d54427d9c17a5e17e6#diff-23c60d19275a3c865f09b7cdc50dbcd8R10